File: baekjoon/2309.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
	int a[9], sum = 0;
	for(int i = 0; i < 9; i++) 
	{
		cin >> a[i];
		sum += a[i];
	}
	
	sort(a, a+9);
	for(int i = 0; i < 8; i++) for(int j = i + 1; j < 9; j++)
	{
		if(sum - a[i] - a[j] == 100)
		{
			for(int k = 0; k < 9; k++)
				if(a[k] != a[i] && a[k] != a[j]) cout << a[k] << endl;
		}
	}
}

File: baekjoon/2309.py
def solve(candidates):
	all_sum = sum(candidates)

	for i in range(8):
		for j in range(i+1, 9):
			exclusive_sum = candidates[i] + candidates[j]
			if all_sum - exclusive_sum == 100:
				winners = []

				for k in range(9):
					if k != i and k != j:
						winners.append(candidates[k])

				winners.sort()

				for winner in winners:
					print(winner)

				return

if __name__ == '__main__':
	candidates = []
	for _ in range(9):
		candidates.append(int(input()))
	solve(candidates)